quesma/quesma/ui/management_console.go

```diff
@@ -261,6 +264,37 @@ func isComplete(value queryDebugInfo) bool {
 	return !reflect.DeepEqual(value.QueryDebugPrimarySource, diag.QueryDebugPrimarySource{}) && !reflect.DeepEqual(value.QueryDebugSecondarySource, diag.QueryDebugSecondarySource{})
 }
 
+func (qmc *QuesmaManagementConsole) GetElasticSearchIndices(ctx context.Context) (indices []string) {
+	sources := qmc.elasticsearchResolveIndexPattern(ctx, "*")
+	for _, index := range sources.Indices {
+		indices = append(indices, index.Name)
+	}
+	return
+}
+
+func (qmc *QuesmaManagementConsole) elasticsearchResolveIndexPattern(ctx context.Context, indexPattern string) (sources elasticsearch.Sources) {
+	resp, err := qmc.elasticsearch.RequestWithHeaders(ctx, "GET", "_resolve/index/"+indexPattern+"?expand_wildcards=open", nil, nil)
```

Comment:
nitpick: The same uri is used in https://github.com/QuesmaOrg/quesma/blob/main/quesma/elasticsearch/index_resolver.go#L56

Maybe, we can extract common const?

Reply:
good idea. 
Overall the whole method could've been extracted but in order to do it really nicely we'd have to ditch that `SimpleClient` in favor of Elasticsearch backend connector which I'd prefer to do after v2 switch